docs/Administration.rst

```diff
@@ -4574,6 +4577,49 @@ The Service Management page is displayed.
    To invoke a method of the selected service, click Run corresponding
    to it. After invoking it successfully, you will see the result
    returned if any.
+   
+   
+.. _ManagingNews:
+
+==============
+Managing News
+==============
+
+How to manage pinned block layouts
+-----------------------------------
+
+The home page CLV is a block that displays articles that have been pinned on the homepage by a user having the role publisher.
+The layout is not set by default, you have to edit the page layout then add a "Content by query" in the area in which you want to set the block.
```

Comment:
I would replace this paragraph by something like:
"The pinned articles are not highlighted by default in the platform. In order to display them in a page, edit the page layout then add a "Content by query" application at the desired location."
And set a link on "pinned articles" to the user guide explaining how to pin an article.

Reply:
updated